source/_posts/2022-01-07-android-quest.markdown

```diff
@@ -0,0 +1,63 @@
+---
+title: "Oculus Quest Meet the Smart Home!"
+description: "Automate your home based on how you use your Quest!"
+date: 2022-01-07 00:00:00
+date_formatted: "January 07, 2022"
+comments: true
+author: Daniel Shokouhi
+categories: Release-Notes
+og_image: /images/blog/2022-01-android-quest/Companion.png
+---
+
+
+![Screenshot of the Android app](/images/blog/2022-01-android/Companion.png)
+
+Hey Everyone! We hope you all had a great new years and holidays spent with the family! This past holiday season had some pretty cool toys going around. Cool toys to us means more things to tinker with. One of the most popular gifts this year was no doubt the Oculus Quest and to be honest how could it not be right? Naturally, working on the Android app some of us wondered how does the app perform on such a device? Well it actually works pretty well and you can start building automations based on your usage today!
+
+<a href="https://sidequestvr.com/app/6427/home-assistant" style="display:inline-block"><img width="200" class="download-badge" alt="Get it on SideQuest" src="https://sidequestvr.com/assets/images/branding/Get-it-on-SIDEQUEST.png" style='box-shadow:none;border:0'></a>
+
+
+## Hello Sidequest!
+
+Wait Sidequest? What is that? Why is this app not on the Oculus Quest Store or even App Lab!? Those are some good questions so lets address why we are on Sidequest first and not in App Lab or the Quest Store itself. To start with we are not a VR ready app, the app found on Sidequest is actually the minimal [version] of the Android app. Why the minimal version? Well the Quest does run Android however, its a pretty heavily modified fork with no Google services. There are no widgets, no shortcuts and does not even allow standard notifications. As some of you may be aware Meta had only recently started to approve of what we qualify as a 2D app. We have begun creating a new version of the app specifically to meet the Quest Store requirements but it will take some time for us to get things to pass inspection. So while we are waiting we didn't want to hold back on releasing something so we can get more feedback and start getting more data for everyone to automate things.
```

Comment:
```suggestion
Wait Sidequest? What is that? Why is this app not on the Oculus Quest Store or even App Lab!? Those are some good questions so let's address why we are on Sidequest first and not in App Lab or the Quest Store itself. To start with we are not a VR-ready app, the app found on Sidequest is actually the minimal [version] of the Android app. Why the minimal version? Well, the Quest does run Android however, it's a pretty heavily modified fork with no Google services. There are no widgets, no shortcuts, and do not even allow standard notifications. As some of you may be aware Meta had only recently started to approve of what we qualify as a 2D app. We have begun creating a new version of the app specifically to meet the Quest Store requirements but it will take some time for us to get things to pass inspection. So while we are waiting we didn't want to hold back on releasing something so we can get more feedback and start getting more data for everyone to automate things.
```